src/main/java/org/primefaces/extensions/component/codescanner/CodeScanner.java

```diff
@@ -60,7 +60,8 @@
         style,
         styleClass,
         width,
-        height
+        height,
+        start
```

Comment:
Can you check what poller calls it?  I think it may be "autoStart"

Reply:
Yes, it is. I'll change it.